### DISCUSSION TITLE
Feature request/suggestion: Better template searching

### DISCUSSION THREAD

**Discussion by LalleSX** [community/community#53249] [Repositories]
### Select Topic Area

Product Feedback

### Body

Searching for templates is a pain as there is not filter option to only have templates shown so there is a lot of repositories that are not template repositories but just have the name template in them. 

So if the search term `template` could be added the same way you search for example programming languages `lang:c++`

then add `type:template` or something like that.

**Comment by LalleSX** [comment]
Agreed.

**Comment by jsoref** [comment]
Isn't this what you're looking for? https://github.com/jsoref?tab=repositories&q=&type=template&language=&sort=
